src/Sylius/Behat/Page/Shop/Checkout/SelectShippingPageInterface.php

```diff
@@ -81,4 +81,14 @@ public function isNextStepButtonUnavailable();
      * @return bool
      */
     public function hasShippingMethod($shippingMethodName);
+
+    /**
+     * @return string
+     */
+    public function getShippingTotal();
+
+    /**
+     * @return string
+     */
+    public function getTotalPrice();
```

Comment:
It's tehnically a BC break, @pamil?

Reply:
There is not BC policy on Behat classes/services, so it should not be a problem ;)